tests/testProcessCcd.py

```diff
@@ -137,19 +137,19 @@ def testProcessCcd(self):
                     print("variance mean = %r, stdDev = %r" % (varMean, varStdDev))
                     print("psf Ixx = %r, Iyy = %r, Ixy = %r" % (psfIxx, psfIyy, psfIxy))
 
-                    self.assertAlmostEqual(bgMean, 191.51595080958367, places=7)
-                    self.assertAlmostEqual(bgStdDev, 0.22492169148323429, places=7)
+                    self.assertAlmostEqual(bgMean, 191.51595080958367, places=3)
+                    self.assertAlmostEqual(bgStdDev, 0.22492169148323429, places=3)
```

Comment:
7 places to 3 places is a huge change! Do differences in software stacks really produce different numbers at the 10^-(3/4) level? If so that is worrying. It implies we might have reproducibility issues in the future. Is this test able to be tightened up to ~5? What are the magnitude of differences we are seeing between versions? Is this related to changing to ints places? Are we losing precision in calculations? I know these are kind of big questions to ask, but making changes to unit tests to make things work scares me unless we can really justify why the change is nessisary.